src/System.Reflection.Metadata/dir.props

```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <PackageVersion>1.4.1</PackageVersion>
+    <PackageVersion>1.4.2</PackageVersion>
     <AssemblyVersion>1.4.1.0</AssemblyVersion>
```

Comment:
This version needs to change too.  Otherwise if the old version is put in the GAC it would win over an updated app-local version.